lib/cli.ts

```diff
@@ -181,7 +181,7 @@ if (!argv.disableChecks) {
   });
 
   if (unknownKeys.length > 0) {
-    throw new Error('Found extra flags: ' + unknownKeys.join(', '));
+    throw new Error('Found extra flags: ' + unknownKeys.join(', ') + ', please use --disableChecks flag to disable the custom flag check!');
```

Comment:
Also, CircleCI does not like your typescript formatting. Please run `npm run format` before submitting.